plugins/cpu/fu-plugin-cpu.c

```diff
@@ -73,6 +75,18 @@ fu_plugin_add_security_attrs_intel_cet (FuPlugin *plugin, FuSecurityAttrs *attrs
 		return;
 	}
 
+	/* check that userspace has been compiled for CET support */
+	if (!g_spawn_command_line_sync (FWUPD_LIBEXECDIR "/fwupd/fwupd-detect-cet",
```

Comment:
I feel like this is asking for a fu_common_get_path type thing.

Reply:
I wondered that too, but do we ever want to override it? Could it just be `g_build_filename (FWUPD_LIBEXECDIR, "fwupd", "fwupd-detect-cet", NULL)` instead?

Reply:
That seems fine to me.

Reply:
Fixed.